.pipelines/cni/pipeline.yaml

```diff
@@ -196,6 +206,38 @@ stages:
       nodeCount: 2
       vmSize: Standard_NC24r
       arch: amd64
+      scaleup: ${SCALEUP_LINUX}
+      iterations: ${ITERATIONS_LINUX}
+
+  - template: singletenancy/linux-cniv2-template.yaml

```

Comment:
Should we just name the file cniv2 template or something ?

Reply:
That is the plan. Just playing nice with ongoing work/rebase.